### ISSUE TITLE
IOS, Profiler fails to connect to the simulator

### ISSUE THREAD

**Issue by cagriy** [JaneySprings/DotNet.Meteor#150]
Hi, this was working just fine 2 days ago, and the only changes I am aware of were the macOS upgrade from 18.1 to 18.2 and the Xcode upgrade from 16.2-beta to 16.2.

If I use the Meteor debugger, it is all fine, it fires up the simulator and runs the app correctly. If I try to use the profiler, either Trace or gcdump, the app fails to start, and I see the following output on the debug console.

Tried with different simulators and different apps (.net 8 and 9), and all have the same issue.

Thanks in advance for any help :)

```
WARNING: dotnet-dsrouter is a development tool not intended for production environments.

info: dotnet-dsrouter-6372[0]
Starting dotnet-dsrouter using pid=6372
info: dotnet-dsrouter-6372[0]
Starting IPC server (/Users/cagri/ios-port.lock) <--> TCP client (127.0.0.1:9000) router.
2024-12-14 11:50:20.165 mlaunch[6373:84274] Requested but did not find extension point with identifier Xcode.IDEDebugger.VariablesViewQuickLookProvider for extension Xcode.SpriteKit.GKStateMachineQuickLookProvider of plug-in com.apple.IDESpriteKitParticleEditor
2024-12-14 11:50:20.166 mlaunch[6373:84274] Requested but did not find extension point with identifier Xcode.IDEDebugger.VariablesViewQuickLookProvider for extension Xcode.IDEDebugger.SpriteKitQuickLookProvider of plug-in com.apple.IDESpriteKitParticleEditor
2024-12-14 11:50:20.171 mlaunch[6373:84274] Requested but did not find extension point with identifier Xcode.IDEPlaygroundsKit.PlaygroundBottomBarExecutionAdaptor for extension Xcode.IDEPlaygroundEditor.IDEPlaygroundBottomBarExecutionAdaptor of plug-in com.apple.dt.IDEPlaygroundEditorPlugin
2024-12-14 11:50:20.172 mlaunch[6373:84274] Requested but did not find extension point with identifier Xcode.DebuggerFoundation.DataSourceConnection for extension Xcode.DebuggerFoundation.watchOS.DataSourceConnectionTargetHub of plug-in com.apple.dt.IDEWatchSupportUI
2024-12-14 11:50:20.172 mlaunch[6373:84274] Requested but did not find extension point with identifier Xcode.Xcode3UI.Xcode3SupportedDestinations for extension IDEWatchSupportUI:IDEWatchSupportUI.plugindata/plug-in[1]/extension[11] of plug-in com.apple.dt.IDEWatchSupportUI
...
...
Please connect the device '1234ABCDE-1234-4D50-1234-1234ABCDE'...
```

**Comment by JaneySprings** [comment]
Hi, @cagriy ! Sorry, but I won't be able to research it until Monday because I don't have any MacOS device at home :)

**Comment by JaneySprings** [comment]
@cagriy It's a bug on my side. Sorry for that. I have fixed this issue in `6.2.2`

**Comment by cagriy** [comment]
@JaneySprings , thanks a lot for the super quick response !
